Packs/Rapid7AppSec/TestPlaybooks/playbook-Rapid7AppSec_Test.yml

```diff
@@ -0,0 +1,1853 @@
+      version: -1
+      name: Verify Outputs
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "true":
+      - "30"
+    separatecontext: false
+    conditions:
+    - label: "true"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Rapid7AppSec.Scan.id
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Rapid7AppSec.Scan.app_id
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Rapid7AppSec.Scan.scan_config_id
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Rapid7AppSec.Scan.submitter_type
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Rapid7AppSec.Scan.submitter_id
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Rapid7AppSec.Scan.submit_time
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Rapid7AppSec.Scan.status
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Rapid7AppSec.Scan.scan_type
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 5800
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "30":
+    id: "30"
+    taskid: 73340ae5-e645-4ea8-8637-72e4f182af4d
+    type: regular
+    task:
+      id: 73340ae5-e645-4ea8-8637-72e4f182af4d
+      version: -1
+      name: app-sec-scan-submit
+      script: rapid7appsec|||app-sec-scan-submit
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "31"
+    scriptarguments:
+      scan_config_id:
+        simple: ${Rapid7AppSec.ScanConfig.id}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 6000
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: a112dd7d-7647-416c-8935-ae6b65de9332
+    type: regular
+    task:
+      id: a112dd7d-7647-416c-8935-ae6b65de9332
+      version: -1
+      name: app-sec-scan-delete
+      script: rapid7appsec|||app-sec-scan-delete
+      type: regular
+      iscommand: true
+      brand: rapid7appsec
+    nexttasks:
+      '#none#':
+      - "32"
+    scriptarguments:
+      scan_id:
+        simple: ${Rapid7AppSec.Scan.id}
```

Comment:
I see that you are using this command after app-sec-scan-list command. 
In order to run this command only with the first scan_id please use the transformer FirstArrayElement.
Please update this in all of the tasks that use such parameters.